testsuite/mixed-domain/pom.xml

```diff
@@ -94,7 +95,7 @@
 
                     <!-- System properties to forked surefire JVM which runs clients. -->
                     <argLine>${jvm.args.ip.client} ${jvm.args.timeouts}</argLine>
-                    <skipTests>false</skipTests>
+                    <skipTests>${ts.skipTests}</skipTests>
```

Comment:
remove skipTests all together.

default values for this attribute behave just fine.